src/Connector.js

```diff
@@ -312,6 +312,8 @@ class Connector {
     });
     await createService(this.iotAgentUrl, this.orionUrl, `/device/${device.id}`, device.id, 'sensor');
     await this.client.subscribe(`/default/${device.id}/cmd`);
+
+    return { id: device.id, token: device.id };
```

Comment:
Please split this commit in two. The first one is related to the URL bug fix. The other one is related to the return of function `addDevice`

Reply:
Also, be more descriptive on commit title. LIke: `Return id and token when adding device` and `Correct URL when verifying existence in IoT Agent`

Reply:
+1

Reply:
Totally agree :smile: 
Thanks for the heads up @ramonhpr !